docs/download.md

```diff
@@ -87,3 +94,39 @@ Here are the steps to set it up:
 Now that you have everything installed, run `npm run build` inside the Leaflet directory.
 This will combine and compress the Leaflet source files, saving the build to the `dist` folder.
 
+### HTML Overview with Leaflet
+
+If choosing to use the hosted version of Leaflet, the example below can demonstrate how to set up the HTML page.
+```
+index.html
+<html>
+  <head>
+    <title>Leaflet Map</title>
+    
+    <!-- Leaflet CSS -->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.8.0/dist/leaflet.css"
+    integrity="sha512-hoalWLoI8r4UszCkZ5kL8vayOGVae1oxXe/2A4AO6J9+580uKHDO3JdHb7NzwwzK5xr/Fs0W40kiNHxM9vyTtQ=="
```

Comment:
Same for integrity hashes — needs `{{site.integrity_hash_css}}` etc.

Reply:
This makes it more complicated because we need to update the docs **after** the release. The hash is only accessible after the push to `unpkg` as far as I know.

Reply:
@Falke-Design it's already used above on this page, it would remain like it's been before.